lib/driver.js

```diff
@@ -5,26 +5,68 @@
 
 const {LDKeyPair} = require('crypto-ld');
 const X25519KeyPair = require('x25519-key-pair');
+const {constants: securityConstants} = require('security-context');
 
 /**
  * Returns a `did:key` method DID Document for a given DID.
  * Note: This is async only to match the async `get()` signature of other
  * `did-io` drivers.
  *
- * @param {string} did
- * @returns {Promise<DidDocument>}
+ * @param {string} did - DID url or a key id (either an ed25519 key or an
+ *   x25519 key-agreement key id).
+ * @param {string} url - alias for the `did` url param, supported for better
+ *   readability of invoking code.
```

Comment:
Is this needed?  I'm not sure it helps readability if there are multiple ways to do the exact same thing.  Where is this called from where "url" makes more sense?

Reply:
@davidlehn Something like this was requested by @dlongley. I think the idea is - since `get()` is responsible for resolving both DID Documents _and_ individual keys, within that document, it may be confusing to developers to see `get({did: keyId})`. And that seeing `get({url: keyId})`, in some cases, may be more enlightening.

Reply:
Ok.  I still don't like it but if it makes sense, then ok.
- Does this need to be strict on the API and fail `if(did && url)`?
- The main method doc should explain this.  Just says "for given DID" now.
- The params should show they are optional as `[did]` and/or `[url]`. Not sure how to best show mutual exclusivity.
- `did` param doc should use caps and have `DID URL` vs `DID url`.

Reply:
@davidlehn Thanks, updated.